db.py

```diff
@@ -12,19 +11,39 @@ class db:
     cursor = None
 
     @staticmethod
-    def GetDBConnection():
+    def getConnection():
         if(db.conn == None):
-            db.conn = pyodbc.connect(os.getenv("Connection_String"))
-
+            BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+            db_path = os.path.join(BASE_DIR, os.getenv("DB_File"))
+            db.conn = sqlite3.connect(db_path, detect_types = sqlite3.PARSE_DECLTYPES | sqlite3.PARSE_COLNAMES)
+        
         return db.conn
 
     @staticmethod
-    def GetDBCursor():
+    def getCursor():
```

Comment:
why the rename? causes more line differences than is necessary

Reply:
I believe the rename helps with clarity and access to the function. I did not think that an increase in line differences is a problem.

Reply:
it's not a problem, but it's an unnecessary change. it's a db cursor whether or not we call it a db cursor. if you want to rename a bunch of function, we can keep that separate to another card. keep prs targeted, makes reviews much more accurate

Reply:
I will leave it for the future then. You are correct about it being unnecessary at the moment.